nss-cohort-36/keahua-arboretum-net-surfers
# Sp having fun

# Description

[Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.]

Closes #[Enter issue #]

## Type of change

Please delete options that are not relevant.



# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout sp-having-fun`
4. do everything and enjoy, non mac users check that your computer doesn't break.

# Checklist:
